pkg/health/condition/check_backup_ready.go

```diff
@@ -37,101 +38,194 @@ const (
 	BackupFailed string = "BackupFailed"
 	// Unknown is a constant that means that the etcd backup status is currently not known
 	Unknown string = "Unknown"
-	// ConditionNotChecked is a constant that means that the etcd backup status has not been updated or rechecked
-	ConditionNotChecked string = "ConditionNotChecked"
+	// NotChecked is a constant that means that the etcd backup status has not been updated or rechecked
+	NotChecked string = "ConditionNotChecked"
```

Comment:
We no longer need this variable at all, since its only usage was when `etcd.Replicas==0`, and we decided to use a different reason string for that. So I'll remove this for now. Even semantically it's ok to do this, since `condition` package sets the conditions only after checking them, so `NotChecked` reason isn't possible anymore.